### DISCUSSION TITLE
eightfish 作为Solana的Layer2

### DISCUSSION THREAD

**Discussion by DaviRain-Su** [creatorsand/.github#14] [Ideas]
项目名: eightfish 
项目介绍: eightfish 作为Solana的Layer2
发起人介绍（社群名）: Mike Tang, Asta
项目地址: https://github.com/eightfish-org/eightfish
参考资料:

**Comment by taokeqin** [comment]
一起整...

**Comment by DaviRain-Su** [comment]
相关资料： 
- https://github.com/solana-labs/solana-program-library/tree/master/account-compression
- https://docs.solana.com/learn/state-compression
- https://github.com/gumhq/gpl
- https://github.com/sgraph-protocol/sgraph/tree/master
- https://github.com/metaplex-foundation/blockbuster
- https://crates.io/crates/spl-account-compression/reverse_dependencies

**Comment by Cobb9527** [comment]
蹲一个
